nix/overlays/haskell.nix

```diff
@@ -1,39 +1,49 @@
-{ gitignore, ghcVersion }:
+{ gitignore, ghc }:
 
 final: prev: {
   haskell = prev.haskell // {
-      packages = prev.haskell.packages // {
-        "${ghcVersion}" = prev.haskell.packages."${ghcVersion}".override {
-          overrides = hfinal: hprev: {
-            # Package Overrides
-            grpc-haskell = final.haskell.lib.dontCheck (hfinal.callPackage ../packages/grpc-haskell.nix { });
-            grpc-haskell-core = final.haskell.lib.doJailbreak (hfinal.callPackage ../packages/grpc-haskell-core.nix { 
-              gpr = final.grpc;
-            });
-            net-mqtt = final.haskell.lib.dontCheck (hfinal.callPackage ../packages/net-mqtt.nix { });
```

Comment:
Looks like the override for `net-mqtt`  got deleted. The build fails for me without it.

Reply:
Yeah.. something weird was happening with opening a shell with `nix develop` and forgot to add this back. Thanks Matt.